tests/ctst/common/common.ts

```diff
@@ -66,3 +78,32 @@ Given('an object {string} that {string}',
             ));
         }
     });
+
+Then('kafka consumed messages should not take too much place on disk',
+    async function (this: Zenko) {
+        const notToCheckTopics = ['oplog', 'dead-letter'];
```

Comment:
i don't think we should ignore `oplog` : it should be cleaned up after it has been consumed...